packages/eds-core-react/src/components/Chip/Chips.stories.tsx

```diff
@@ -3,29 +3,27 @@ import styled from 'styled-components'
 import { Icon, Chip, ChipProps, Avatar, AvatarProps } from '../..'
 import { Meta, Story } from '@storybook/react'
 import { save } from '@equinor/eds-icons'
+import { Stack as SBStack } from './../../../.storybook/components'
+import page from './Chips.docs.mdx'
 
 const icons = {
   save,
 }
 
 Icon.add(icons)
 
-const Wrapper = styled.div`
+const Stack = styled(SBStack)`
```

Comment:
Not sure if there is much point extending the "Stack" component when pretty much everything is overwritten anyway? Feels like it might make it easier to break stuff if something is changed in stack, now you have to check every place that partially extended it, so its bad for maintainability